### PR TITLE
Build redirector images.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: redirector/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,65 @@
+#
+# This file is autogenerated by pip-compile
```
Comment (pmac) [inline redirector/requirements.txt:2]: pip-tools FTW!

**Review by pmac — APPROVED** [review]
🦃

File: redirector/requirements.txt (hunk 1 above)
Comment (glogiotatidis) [reply redirector/requirements.txt:2]: yes, they are pretty neat. I got the tip from you in basket, thanks!

**Comment by glogiotatidis** [comment]
Superseeded by #1478

**Pull request closed without merging**